recipe/meta.yaml

```
@@ -1,22 +1,27 @@
 {% set name = "sparrow" %}
-{% set version = "0.0.1" %}
+{% set version = "0.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/xtensor-stack/sparrow/archive/{{ version }}.tar.gz
```

`home` and `dev_url` in the `about` section also need to be changed.

```suggestion
  url: https://github.com/man-group/sparrow/archive/{{ version }}.tar.gz
```